src/dashboard/index.js

```diff
@@ -1,42 +1,353 @@
+      document.querySelector("#content").innerHTML = text;
+    });
+}
 
 function render() {
   document.getElementById("repos").addEventListener("click", () => {
-    const repository = path.join("file://", __dirname, "repos.html");
+    document.getElementById("chef-cookbook-content").innerHTML = "";
+    document.getElementById("chef-repo-content").innerHTML = "";
+    document.getElementById("overview-content").style.display = "none";
+    const table = ` <table class="table table-bordered" id="table-repos">
+<thead>
+  <tr>
+    <th>
+      <div class="custom-control custom-checkbox">
+        <input
+          type="checkbox"
+          class="custom-control-input"
+          id="customCheck2"
+        />
+      </div>
+    </th>
+    <th scope="col">Repositories</th>
+  </tr>
+</thead>
+</table>`;
+
+    document.getElementById("chef-repo-content").innerHTML += table;
+
+    axios(
+      "http://localhost:7050/api/v1/repositories/list_repositories?page=1&limit=10",
+      {
+        headers: {
+          Authorization:
+            "eyJhbGciOiJIUzI1NiJ9.eyJhY2Nlc3Nfa2V5IjoiZTY3NmEyYjI1MTg4ZGU2NTBiZWQyMDVkYWQxNzI0MDMiLCJleHAiOjE2Njg4MDE4OTF9.YyK57cUKFmJfgQzQwczpyEDf1s1ud0Wi69xfO23K5fQ",
+        },
+      }
+    ).then(function (res) {
+      const data = res.data.repositories;
+      document.getElementById("chef-repo-content").style.display = "block";
+      let tablecontents = "";
+      for (var i = 0; i < data.length; i++) {
+        tablecontents += "<tr>";
+        const rows = Object.values(data[i]);
+        tablecontents += `<td>
+        <div class="custom-control custom-checkbox">
+        <input type="checkbox" class="custom-control-input" id="customCheck2">
+    </div>
+    </td>`;
+        tablecontents += "<td>" + rows[2] + "</td>";
+        // tablecontents += "<td>" + rows[2] + "</td>";
+
+        tablecontents += "</tr>";
+      }
+      document.getElementById("table-repos").innerHTML += tablecontents;
+    });
+  });
+
+  document.getElementById("cookbooks").addEventListener("click", () => {
+    document.getElementById("chef-cookbook-content").innerHTML = "";
+    document.getElementById("overview-content").style.display = "none";
+    var cols = document.getElementsByClassName("content");
+    for (i = 0; i < cols.length; i++) {
+      cols[i].style.display = "none";
+    }
+    document.getElementById("chef-cookbook-content").style.display = "block";
+    const table = ` <table class="table table-bordered" id="table-cookbooks">
+        
+<thead>
+  <tr>
+    <th>
+      <div class="custom-control custom-checkbox">
+        <input
+          type="checkbox"
+          class="custom-control-input"
+          id="customCheck2"
+        />
+      </div>
+    </th>
+    <th scope="col">Cookbooks</th>
+    <th scope="col">Repository</th>
+  </tr>
+</thead>
+</table>`;
+
+    document.getElementById("chef-cookbook-content").innerHTML += table;
+
+    axios("http://localhost:7050/api/v1/repositories/cookbooks", {
```

Comment:
@nikhil2611 
Axios comes with a concept called [interceptors](https://github.com/axios/axios#interceptors) to intercept any requests and add authorization headers.

Via using the thing, one don't have to append the auth token every time. We can take it as an improvement also